themes/classic/modules/ps_linklist/views/templates/hook/linkblock.tpl

```diff
@@ -37,7 +37,7 @@
           </span>
         </span>
       </div>
-      <ul id="footer_sub_menu_{$_expand_id}" class="collapse">
+      <ul id="footer_sub_menu_{$_expand_id}" {if $linkBlock.hook == 'displayFooter' }class="collapse"{/if}>
```

Comment:
If there is a different behavior depending on hook, it's better to use two templates. We absolutely want to avoid if statement in templates